src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.dao.FruitShopDao;
+import core.basesyntax.dao.FruitShopDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction fruitTransaction) {
+        FruitShopDao fruitShopDao = new FruitShopDaoImpl();

```

Comment:
make it global and initialize it using the constructor, the same for other handlers